pkg/hook/hook.go

```diff
@@ -55,8 +57,19 @@ func (h *Hook) Dump() ([]byte, error) {
 	return yaml.Marshal(h)
 }
 
+func (h *Hook) joinURLParams() string {
+	params := make([]string, 0, len(h.Params))
+	for k, v := range h.Params {
+		params = append(params, fmt.Sprintf("%v=%v", k, v))
+	}
+	return "?" + strings.Join(params, "&")
+}
+
 // Fire sends an HTTP request to the given target.
 func (h *Hook) Fire(target string) (*http.Response, error) {
+	if len(h.Params) >= 1 {
+		target += h.joinURLParams()
```

Comment:
Set this in `h.toRequest` instead. e.g. something along the lines of https://play.golang.org/p/YgRtsfSVtE5